extensions/amp-story/1.0/page-advancement.js

```diff
@@ -520,7 +520,7 @@ export class ManualAdvancement extends AdvancementConfig {
 
     if (
       this.isInStoryPageSideEdge_(event, pageRect) ||
-      this.isTooLargeOnPage_(target, pageRect)
+      this.isTooLargeOnPage_(event, target, pageRect)
```

Comment:
For future reference, we could just pass the event and extract the target from there, instead of having to pass both the event and the target.